docs/deep-dive/02-dd-create-git-function-serverless.md

```diff
@@ -120,64 +120,53 @@ Follows these steps:
 
     </details>
     <details>
-    <summary label="console-ui">
-    Console UI
+    <summary label="busola-ui">
+    Busola UI
     </summary>
 
+>**NOTE:** Busola is not installed by default. Follow the [instructions](https://github.com/kyma-project/busola/blob/main/README.md) to install it with npx.
+
 1. Create a Namespace or select one from the drop-down list in the top navigation panel.
 
 2. Create a Secret (optional).
 
   If you use a secured repository, you must first create a Secret with either basic (username and password or token) or SSH key authentication to this repository in the same Namespace as the Function. To do that, follow these sub-steps:
 
-  - On your machine, create this YAML file with one of these Secret definitions:
+  a. Open your Namespace view. In the left navigation panel, go to **Configuration** > **Secrets** and select the **Create Secret** button.
 
-    - Basic authentication:
+  b. In the **Metadata** tab, enter the Secret name under **Name**.
 
-      ```yaml
-      apiVersion: v1
-      kind: Secret
-      metadata:
-        name: git-creds-basic
-        namespace: {NAMESPACE}
-      type: Opaque
-      data:
-        username: {USERNAME}
-        password: {PASSWORD_OR_TOKEN}
-      ```
+  c. In the **Data** tab, select **Add data entry** and enter these key-value pairs with credentials:
 
-    - SSH key:
+    - Basic authentication:
 
+    ```bash
+    username: {USERNAME}
+    password: {PASSWORD_OR_TOKEN}
+    ```
 
-      ```yaml
-      apiVersion: v1
-      kind: Secret
-      metadata:
-        name: git-creds-key
-        namespace: {NAMESPACE}
-      type: Opaque
-      data:
-        key: {SSH_KEY}
-      ```
+   - SSH key:
 
-    >**NOTE:** Read more about the [supported authentication methods](#details-git-source-type).
+    ```bash
+    key: {SSH_KEY}
+    ```
 
-  - Go to your Namespace view and select **Deploy new workload** > **Upload YAML**.
+   >**NOTE:** Read more about the [supported authentication methods](#details-git-source-type).
 
-  - Locate the YAML file with the Secret and select **Deploy**.
+   - Confirm by selecting **Create**
```

Comment:
Shouldn't it be rather the d) point instead of another bullet point?

Reply:
Good catch!